static/src/javascripts/projects/common/modules/commercial/dfp/render-advert.js

```diff
@@ -48,8 +48,8 @@ define([
      */
     sizeCallbacks[adSizes.mpu] = function (_, advert) {
         if (advert.node.classList.contains('ad-slot--right')) {
-            var mobileAdSizes = advert.sizes('data-mobile');
-            if (mobileAdSizes && mobileAdSizes.indexOf([300, 251]) > -1) {
+            var mobileAdSizes = advert.sizes.mobile;
+            if (mobileAdSizes && mobileAdSizes.some(function (size) { return size[0] === 300 && size[1] === 251; })) {
```

Comment:
Yes, very good idea. Ultimately everything that concerns ad sizes should be exposed through the `adSizes` API.